src/stock_str.rs

```diff
@@ -419,6 +419,9 @@ pub enum StockMessage {
 
     #[strum(props(fallback = "Member %1$s added."))]
     MsgAddMember = 173,
+
+    #[strum(props(fallback = "⚠️ Your email provider does not allow sending unencrypted emails"))]
```

Comment:
i think, the final string should be more on-point and should also show a way to fix - but that could be done in the UI that anyways overwrites this string.

my suggestion would be sth. as 

>  End-to-end-encryption required by nine.testrun.org but not set up in this chat. Tap to learn how to fix this.

"Learn more" would then show a dialog with some more help and buttons to open help - and to set up end-to-end-encryption, offering QR code

showing the provider's domain may be nice at it help son confusion wrt multi-account and also the user knows whom to blame :)

for this PR, this would mean that %1$s is being replaced by provider's domain